src/startupscriptgenerator/src/common/Gopkg.toml

```diff
@@ -41,6 +41,10 @@
   name = "github.com/spf13/afero"
   version = "=1.8.2"
 
+[[override]]
+  name = "golang.org/x/sys"
```

Comment:
Isn't this coming from Paul's latest PR? Do you need to rebase, or are you adding this to other files?

Reply:
I merge Paul's branch `bugfix/lock-sys-unix-package` with my branch since I also need his changes and since they're not in `main` yet